app/Locale/ChineseSimplified.php

```diff
@@ -297,6 +297,7 @@ public function get_locale(): array
 			'ERROR_SEARCH_DEACTIVATED' => '搜索功能已在设置中停用。',
 			'SUCCESS' => 'OK',
 			'RETRY' => '重试',
+			'OVERRIDE' => 'Override',
```

Comment:
I still have to check with the frontend where we use this label, but in order to prevent clash of names in the future I would prefer a key which is more specific than only `OVERRIDE`. I would expect something like `CHECKBOX_OVERRIDE_LABEL` or whatever is appropriate. (I haven't had a look at the frontend yet, hence I don't know where this key is used.)